components/WebinarTemplate/WebinarTemplate.tsx

```diff
@@ -52,59 +60,85 @@ export const WebinarTemplate = ({
           >
             {title}
           </Box>
-          <Flex
-            fontSize={["text-lg", "text-xl"]}
-            lineHeight={["md", "lg"]}
-            color="dark-gray"
-            my={2}
-            flexWrap="nowrap"
-          >
-            <Flex pr={2} fontWeight="bold" color="darkest">
-              Date:
-            </Flex>
-            {date}
-          </Flex>
-          <Flex
-            fontSize={["text-lg", "text-xl"]}
-            lineHeight={["md", "lg"]}
-            color="dark-gray"
-            my={2}
-            flexWrap="nowrap"
-          >
-            <Flex pr={2} fontWeight="bold" color="darkest">
-              Time:
-            </Flex>
-            {time}
-          </Flex>
 
-          {/* speaker section: display multiple speakers in a column format/single speaker as a row*/}
-          <Flex
-            fontSize={["text-lg", "text-xl"]}
-            lineHeight={["md", "lg"]}
-            color="dark-gray"
-            mt={2}
-            mb={multipleSpeakers ? 4 : 2}
-            flexDirection={multipleSpeakers ? "column" : "row"}
-          >
-            {multipleSpeakers ? (
-              <>
+          {/* Image and Webinar Details section */}
+          <Flex flexDirection={["column", "row"]} my={[4, 6]}>
+            {/* image if available */}
+            {image && (
+              <Flex justifyContent="flex-start">
+                <Flex
+                  width="370px"
```

Comment:
```suggestion
                  width={multipleSpeakers ? 370 : 202}
```
Merely an example, haven't run the code.